tests/cuda/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2021-2022 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+include(traccc-compiler-options-cuda)
+
```

Comment:
Please add

```cmake
find_package( CUDAToolkit )
```

into this file. It gets called upstream, making `CUDA::cudart` visible at this point. But we should not rely on that. (I don't even fully understand right now why that imported target is actually visible here... :confused:)